src/bindist/config.sys

```diff
@@ -9,6 +7,6 @@ buffers=10
 devicehigh=e:\dosemu\ems.sys
 devicehigh=e:\dosemu\cdrom.sys
 rem uncomment to load another bitmap font (see also autoemu.bat)
-rem devicehigh=c:\dos\display.sys con=(ega,,1)
+rem devicehigh=d:\dos\display.sys con=(ega,,1)

```

Comment:
That works for this line.
The lines in autoemu.bat are less trivial though. I had to change:
mode con codepage select 850 to mode con codepage select=850 (add a =) to let it work
But the line chcp 850, just gives "NLSFUNC not installed".

I don't know if we really need these lines now? I only have experience with English and Dutch DOS versions which do not really go much beyond ASCII. I would imagine we'd read out the LANG environment variable from the system at some point so that we can pick the codepage which makes the most sense for a user.